common/djangoapps/util/milestones_helpers.py

```diff
@@ -9,13 +9,19 @@
 from opaque_keys import InvalidKeyError
 from opaque_keys.edx.keys import CourseKey
 
+from milestones import api as milestones_api
```

Comment:
I would add that if you do find circular import issues, it would still be good to disentangle them if you can, i.e., move common code out to a separate file that both milestones_helpers and milestones.api can depend on.  Only if that proves too difficult should you leave the function level imports.